backend/README.md

```diff
@@ -0,0 +1,4 @@
+**This is a readme for Backend component of the React Product Component App.**
```

Comment:
Could u improve this README?

Reply:
I'll add some formatting and information about .env file that has to have a PORT environmental variable defined